controllers/user_profilesControllers.js

```diff
@@ -49,10 +53,28 @@ const updateUserProfile = async (id, data) => {
   }
 };
 
+const uploadProfileAvatar = async (id, image) => {
+  try {
+    const newImage = await Image.query().insert(image);
+    const image_id = newImage.id;
+    const obj = { avatar: image_id };
+    const uploadProfileAvatar = await UserProfile.query().patchAndFetchById(
```

Comment:
I think it will be better to rename this variable to a better name make code more readable, because function is called `uploadProfileAvatar` too.